packages/server/lib/browsers/index.coffee

```diff
@@ -32,30 +32,24 @@ cleanup = ->
 
 getBrowser = (name) ->
   switch name
-    ## normalize all the chrome* browsers
-    when "chrome", "chromium", "canary"
-      require("./chrome")
     when "electron"
       require("./electron")
+    else
+      require("./chrome")
```

Comment:
whenever we add more browsers this will have to change back to what it was

Reply:
yep. made it fall-through for now so we can add more editions of Chrome without having to edit this file - `chrome-beta` is the one I'm thinking of in particular